bindata/v3.11.0/kube-apiserver/recovery-pod.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: openshift-kube-apiserver
+  name: kube-apiserver-recovery
+  labels:
+    revision: "recovery"
+spec:
+  containers:
+  - name: kube-apiserver-recovery
+    image: "{{ .KubeApiserverImage }}"
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["hypershift", "openshift-kube-apiserver"]
+    args:
+    - --config=/etc/kubernetes/static-pod-resources/config.yaml
```

Comment:
this is the same config as the static pod? How does this work with 7443 below? Would have expected `recovery-config.yaml`.

Reply:
saw the config. I guess the config.yaml will be the recovery config at the end with the recovery dir as the `static-pod-resources`.

Reply:
it is a different one
https://github.com/openshift/cluster-kube-apiserver-operator/blob/3a5f313893a373f7b3f766d3c08ef17ce6f19fed/bindata/v3.11.0/kube-apiserver/recovery-config.yaml#L1
https://github.com/openshift/cluster-kube-apiserver-operator/blob/3a5f313893a373f7b3f766d3c08ef17ce6f19fed/pkg/recovery/apiserver.go#L260-L264